server/apps/cassandra-app/sample-configuration/jmap.properties

```diff
@@ -24,7 +24,7 @@ tls.secret=james72laBalle
 # view.email.query.enabled=true
 
 # If you want to specified authentication strategies for Jmap draft version
-# authentication.strategy.draft=org.apache.james.AuthStratA,org.apache.james.AuthStratB
+# authentication.strategy.draft=org.apache.james.jmap.http.AccessTokenAuthenticationStrategy,org.apache.james.jmap.jwt.JWTAuthenticationStrategy,org.apache.james.jmap.http.QueryParameterAccessTokenAuthenticationStrategy
 
 # If you want to specified authentication strategies for Jmap rfc-8621 version
-# authentication.strategy.rfc8621=org.apache.james.AuthStratA,org.apache.james.AuthStratB
\ No newline at end of file
+# authentication.strategy.rfc8621=org.apache.james.jmap.jwt.JWTAuthenticationStrategy,org.apache.james.jmap.http.BasicAuthenticationStrategy
```

Comment:
Can we rely on an implicit FQDN prefix to simplify this?

```
# authentication.strategy.draft=AccessTokenAuthenticationStrategy,JWTAuthenticationStrategy,QueryParameterAccessTokenAuthenticationStrategy

# If you want to specified authentication strategies for Jmap rfc-8621 version
# authentication.strategy.rfc8621=JWTAuthenticationStrategy,BasicAuthenticationStrategy 
```

?

We can easily use `NamingScheme` `OptionalPackagePrefix` alongside with `GuiceGenericLoader`, use `org.apache.james.jmap.http` as an optional prefix, and also move `JWTAuthenticationStrategy` to `org.apache.james.jmap.http` package.